src/main/java/org/mitre/inferno/Validator.java

```diff
@@ -86,6 +100,64 @@ public void loadProfile(Resource profile) {
     context.cacheResource(profile);
   }
 
+  private List<String> getProfileUrls(String id) throws IOException {
+    String version = null;
+    if (id.contains("#")) {
```

Comment:
NIT: might be neater to accomplish this with something like `id.split("#")`, where `id.split("#")[0]` is the ID, and `id.split("#")[1]` is the version